dockerOS.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
can we use a speaking name for this file? the others often imply what it does, not only what platform it aims at (eg. `groupDevDependencies`), which is really nice

maybee `autoMergeNonMajorDocker`